### ISSUE TITLE
Tileserver doesn't start

### ISSUE THREAD

**Issue by luandro** [digidem/edt-offline#31]
Tile-server should have a default tile-set, currently it throws error:

```
Starting tileserver-gl v4.4.10
No MBTiles specified, using zurich_switzerland.mbtiles
[INFO] Automatically creating config file for zurich_switzerland.mbtiles
[INFO] Only a basic preview style will be used.
[INFO] See documentation to learn how to create config.json file.
ERROR: Metadata missing in the MBTiles.
Make sure zurich_switzerland.mbtiles is valid MBTiles.
```

**Comment by luandro** [comment]
```
demo-tileserver-1    | Moving files to /data folder
demo-tileserver-1    | /usr/src/app/docker-entrypoint.sh: 27: git: not found
demo-tileserver-1    | cp: cannot stat 'tmp-dir/fonts/*': No such file or directory
demo-tileserver-1    | cp: cannot stat 'tmp-dir/sprites/*': No such file or directory
demo-tileserver-1    | cp: cannot stat 'tmp-dir/styles/*': No such file or directory
```

**Comment by luandro** [comment]
Solved with #36
